tests/baselines/reference/tsserver/projects/file-opened-is-in-configured-project-that-will-be-removed.js

```diff
@@ -406,42 +406,18 @@ Info seq  [hh:mm:ss:mss] event:
         "diagnostics": []
       }
     }
-Info seq  [hh:mm:ss:mss] `remove Project::
+Info seq  [hh:mm:ss:mss] getConfigFileNameForFile:: File: /user/username/projects/myproject/playground/tsconfig-json/tsconfig.json ProjectRootPath: undefined:: Result: /user/username/projects/myproject/playground/tsconfig.json
 Info seq  [hh:mm:ss:mss] Project '/user/username/projects/myproject/playground/tsconfig.json' (Configured)
 Info seq  [hh:mm:ss:mss] 	Files (4)
```

Comment:
I assume the changes here are positive, in that it's finding another reasonable project for the file?

Reply:
(Not 100% sure what "will be removed" means for these tests; this folder has a couple of tests that change like this.)

Reply:
Earlier we were removing the project from memory that means free up sources, but now the file is part of the project 
Info seq  [hh:mm:ss:mss] 	FileName: /user/username/projects/myproject/playground/tsconfig-json/tests/spec.ts ProjectRootPath: undefined
Info seq  [hh:mm:ss:mss] 		Projects: /user/username/projects/myproject/playground/tsconfig.json